airflow/config_templates/default_airflow.cfg

```
@@ -468,6 +468,9 @@ cookie_secure = False
 # Set samesite policy on session cookie
 cookie_samesite =
 
+# Set session cookie name
+cookie_name =
```

Let me know if you'd prefer:
```suggestion
cookie_name = session
```